app/mesh/1.2.x/features/opa.md

```diff
@@ -18,9 +18,13 @@ When `OPAPolicy` is applied, the control plane configures:
 To apply a policy with OPA:
 
 - Specify the group of data plane proxies to apply the policy to with the `selectors` property.
-- Provide the list of policies with the `conf` property. Policies are defined in the [Rego language](https://www.openpolicyagent.org/docs/latest/policy-language/).
+- Provide a policy with the `conf` property. Policies are defined in the [Rego language](https://www.openpolicyagent.org/docs/latest/policy-language/).
+{:.note}
+> **Note:** You cannot currently apply multiple OPA policies. This limitation will be addressed in the future.
```

Comment:
Should we just link to the OPA issue? (Makes it clear it's not an us limitation :) )

Reply:
Jakub suggested not to do that at this moment - https://github.com/Kong/kong-mesh/pull/2516#issuecomment-1404876931

Reply:
Ok then